web/frontend/src/components/arm.vue

```diff
@@ -228,7 +228,7 @@ const armCopyJoints = (status: ArmStatus) => {
       />
     </div>
     <div class="border border-t-0 border-black p-4">
-      <div class="flex flex-wrap gap-4 mb-4">
+      <div class="mb-4 flex flex-wrap gap-4">
```

Comment:
Most linting changes in this PR are due to Tailwind's ESLint plugin changing preferred class order.